src/components/Common/Forms.js

```diff
@@ -71,3 +82,102 @@ export const Fieldset = styled.section`
     margin-bottom: ${remcalc(24)};
   }
 `;
+
+const fadeIn = keyframes`
+  from{
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+export const SearchField = styled.section`
+  position: relative;
+  margin-bottom: ${remcalc(36)};
+`;
+
+export const Results = styled.ul`
+  width: 100%;
+  max-height: ${remcalc(200)};
+  list-style-type: none;
+  background: ${props => props.theme.colors.greyBg};
+  position: absolute;
+  left: 0;
+  top: ${remcalc(56)};
```

Comment:
56px is off from our spacing system... shouldn't we use 54px -> `theme.space[4]` ?